src/System.Numerics.Vectors/ref/System.Numerics.Vectors.csproj

```diff
@@ -3,7 +3,10 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
     <ProjectGuid>{650277B5-9423-4ACE-BB54-2659995B21C7}</ProjectGuid>
-    <IsPartialFacadeAssembly Condition="'$(TargetGroup)' == 'netfx' OR '$(TargetGroup)' == 'net46'">true</IsPartialFacadeAssembly>
+    <IsTargetingNetFx Condition="'$(TargetGroup)'=='netfx' OR '$(TargetGroup)'=='net46'">true</IsTargetingNetFx>
```

Comment:
Do we need this condition?

Can we just collapse this into the IsPartialFacadeAssembly and use that?

```csproj
<IsPartialFacadeAssembly Condition="'$(TargetGroup)'=='netfx' OR '$(TargetGroup)'=='net46'">true</IsPartialFacadeAssembly>
```

Reply:
`IsTargetingNetFx` is also used at line 28 below. Hence made the changes to `IsPartialFacadeAssembly` to consolidate the conditions in one place.

Reply:
These properties mirror what is in `src\System.Numerics.Vectors.csproj`

https://github.com/dotnet/corefx/blob/9edb1251fd40c1cc75e80cd4f0b3659b1de26aa4/src/System.Numerics.Vectors/src/System.Numerics.Vectors.csproj#L9-L12

I think it is clearer to define them this way, so you can write code later that says:

"If targeting netfx, set this or that"

Instead of code that says:

"If is partial facade assembly, set this or that"

Then you need to constantly look up and say "When is this a partial facade?".

Reply:
> I think it is clearer to define them this way

Good point. I am convinced.